### DISCUSSION TITLE
How to get out of Pluto.ji

### DISCUSSION THREAD

**Discussion by woodsjay** [fonsp/Pluto.jl#864] [Questions about using Pluto]
As a total noob, the simplest things are the hard. You need to work as hard at getting people out as you do at attracting people in. 

In the transition from installing the package back to Julia - how is this done?

(@v1.5) pkg> c/r

(@v1.5) pkg> exit()
ERROR: Could not determine command

(@v1.5) pkg> ctrl d, ?

I solved the problem by bailing out of Julia altogether. There must be a better way. Please put it in the "Simple" document before the next step of entering the julia commands.

**Comment by icweaver** [comment]
Hey @woodsjay! I don't think this is so much a Pluto specific problem as it is a general usage question. The `(@v1.5) pkg>` prompt you mentioned is part of julia's package manager, which is separate from Pluto, and is used to install other packages in the julia ecosystem as well.

To hop out of the package manager and back to the main "Julian mode", you can just hit backspace. To hop back in, just hit `]`. More cool modes available in the REPL can be found [here](https://docs.julialang.org/en/v1/stdlib/REPL/#The-different-prompt-modes)
